cli/src/commands/upload.ts

```diff
@@ -68,7 +68,8 @@ export class Upload extends BaseCommand {
       }
     }
 
-    const existingAlbums = (await this.immichApi.albumApi.getAllAlbums()).data;
+    const allAlbums = await this.immichApi.albumApi.getAllAlbums();
+    const existingAlbums = allAlbums.data;
```

Comment:
```suggestion
    const { data: existingAlbums } = await this.immichApi.albumApi.getAllAlbums();
```
Does this work instead for these types of variables?